cmake/modules/falcosecurity-libs-repo/patches/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2023 The Falco Authors.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+file(GLOB patches "*.patch")
+file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/libs-patches/)
+foreach(p ${patches})
+    configure_file("${p}" "${PROJECT_BINARY_DIR}/libs-patches/" @ONLY)
```

Comment:
libelf patch uses `@CMAKE_SYSTEM_PROCESSOR@` thus it needs to be configured.
`@ONLY` to avoid touching also normal cmake variables like `${FOO}`.